locales/zh-TW.js

```diff
@@ -0,0 +1,628 @@
+Snipcart.execute('registerLocale', 'zh-TW', {
+yes:
+"是",
+no:
+"没有",
+print:
+"打印",
+download_as_pdf:
+"以PDF格式下載",
+checkout:
+"結帳",
+close:
+"關閉",
+first_name:
+"姓",
+name:
+"名字",
+last_name:
+"名字",
+company_name:
+"公司名稱",
+share_by_email:
+"分享到電子信箱",
+email:
+"電子信箱",
+password:
+"密碼",
+confirm_password:
+"確認密碼",
+ok:
+"確認",
+send:
+"寄送",
+address_1:
+"街道地址",
+address_2:
+"街道地址2",
+city:
+"城市",
+postal_code:
+"郵政編碼",
+phone:
+"電話",
+previous:
+"上一步",
+next:
+"下一步",
+finalize:
+"完成",
+country:
+"國家",
+subtotal:
+"小計",
+rebate:
+"折扣",
+apply_promo_code:
+"申請促銷代碼",
+my_cart:
+"我的購物車",
+my_cart_content:
+"我的購物車內容",
+shipping_method:
+"購物方式",
+payment_method:
+"付款方式",
+confirm_order:
+"確認訂單",
+bill_me_later:
+"一會兒再結帳",
+bill_me_later_action:
+"請稍後為您撥打電話",
+bill_me_later_explanation:
+"發票將電子郵件發送給您",
+pay_via_mollie:
+"選擇付款方式",
```

Comment:
[Mollie](https://www.mollie.com/) is a brand, I would expect it to look a bit like Paypal. Could you validate? 🙂

Reply:
Here, in Taiwan, they have a name for it. If you prefer, you can change it but not sure taiwanese will understand